### PR TITLE
[p5.js hi ]Added image and typography

### PR DESCRIPTION
**Pull request by Surbhi-Pittie**

### CODE REVIEW

File: src/content/reference/hi/p5/createImage.mdx [hunk 1]
```diff
@@ -0,0 +1,169 @@
+---
+title: createImage
+module: छवि
+submodule: छवि
+file: src/image/image.js
+description: >
+  <p>एक नया <a href="/reference/p5/p5.Image">p5.Image</a> ऑब्जेक्ट बनाता है।</p>
+
+  <p><code>createImage()</code> <code>width</code> का उपयोग करता है और
+  <कोड>ऊंचाई</कोड> पैरामीटर नया सेट करने के लिए
```
Comment (Divyansh013) [inline src/content/reference/hi/p5/createImage.mdx:10]: don't translate code

File: src/content/reference/hi/p5/get.mdx [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+title: get
+module: छवि
+submodule: पिक्सल
+file: src/image/pixels.js
+description: >
+  <p>कैनवास से एक पिक्सेल या पिक्सेल का एक क्षेत्र प्राप्त करता है।</p>
+
+  <p><code>get()</code> का उपयोग करना आसान है लेकिन यह उतना तेज़ नहीं है
+
+  <a href="/reference/p5/पिक्सेल">पिक्सेल</a>। <ए का प्रयोग करें
```
Comment (Divyansh013) [inline src/content/reference/hi/p5/get.mdx:11]: anchor tag should be as it is

**Review by Divyansh013 — CHANGES_REQUESTED** [review]

**Review by Divyansh013 — APPROVED** [review]

**Comment by Divyansh013** [comment]
@TanviKumar PTAL !

**Review by TanviKumar — APPROVED** [review]

**Comment by TanviKumar** [comment]
@Qianqianye the PR can be merged

**Comment by Qianqianye** [comment]
Thanks @Surbhi-Pittie @Divyansh013 @TanviKumar!

**Pull request merged**